### PR TITLE
Header headline section

### PR DESCRIPTION
**Pull request by mirouhml**
- Add toolbar with title and menu button.
- Add headline section that has 4 parts: primary text, supporting text, description and social media buttons.
- Add background to the headline section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <div class="header-tab">
```
Comment (ReshyResh) [inline index.html]: - [x] The header should be wrapped inside a `<header>` tag, both for both for semantic and search engine optimizazion reasons.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        </div>
+        <div class="header-content">
+            <div class="header-primary-text">
+                Hey There. <br> I'm Ammar
+            </div>
+            <div class="header-supporting-text">
+                I'm a Software Developer
+            </div>
+            <div class="header-description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (ReshyResh) [inline index.html]: - [x]  Text should be wrapped inside a paragraph `<p>` tag, same reasons as above.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            </div>
+            <div class="header-social-media">
```
Comment (ReshyResh) [inline index.html:35]: - [x]  This is an unordered list, it is required to use the `<ul>` tag for the list and `<li>` for its elements, which should also redirect to your social links, meaning they should have an anchor tag (`<a>`).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            <div class="tab-title">
+                Ammar Hamlaoui
+            </div>
+            <div class="menu-button">
+                <img src="src/union.png" alt="Menu icon"/>
+            </div>
+        </div>
+        <div class="header-content">
+            <div class="header-primary-text">
+                Hey There. <br> I'm Ammar
```
Comment (ReshyResh) [inline index.html]: - [x]  This is the main title and it should be wrapped inside an `<h1>` tag, same reason as above. Kindle make sure to use semantic HTML as much as you can!

File: index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            </div>
+            <div class="header-supporting-text">
+                I'm a Software Developer
```
Comment (ReshyResh) [inline index.html]: - [x]  Secondary titles should be wrapped in `<h2>` or `<h3>` tags, for the same reasons as above.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @mirouhml , 

### You did a really good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:
  - ✔ Descriptive pull request.
  - ✔ Meaningful commit messages.
  - ✔ All content is aligned with flexbox.
  - ✔ The design matches the Figma template


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### None.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mirouhml) [reply index.html]: Is the header the toolbar only or the whole page I made?

Comment (ReshyResh) [reply index.html]: > Is the header the toolbar only or the whole page I made?

It is only the first section

Comment (mirouhml) [reply index.html]: Okay thank you.

**Review by leolpaz — APPROVED** [review]
Hi @mirouhml , 

Your project is complete!
Congratulations! 🎉🎉 

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**